### ISSUE TITLE
Restriction of needing to ensure only 1 action is running at a time when mountPropagation workaround is being used.

### ISSUE THREAD

**Issue by jwmatthews** [konveyor/mig-controller#156]
This issue is to ensure we track future concerns of running multiple actions in parallel.  By multiple actions, I mean if we have multiple MigrationPlans, in future we will likely want to allow running Stage on each MigrationPlan at same time, assuming they do not overlap, no shared namespaces.  We *will* continue to want to restrict inside of a MigrationPlan queueing actions and running them sequentially on a per MigrationPlan basis.

So...assuming we multiple MigrationPlans that do not overlap.
In future, we would like to be able to have say a Stage running potentially on each MigrationPlan at same time.

If we did this with our mountPropagation workaround we'd have potential issues as we restart the Restic daemonset  on a per action run basis.

I don't believe this a concern as of today, my understanding is that Velero is limiting actions run on a cluster to 1 action only running cluster wide, with future work to allow running multiple actions concurrently.

If Velero begins to support running multiple actions concurrently we will want to revisit the mountPropagation workaround, or perhaps continue to enforce a sequential/serial run of actions globally.

**Comment by djwhatle** [comment]
Migrations are limited to one at a time by the `Suspended` mechanism.
